pkg/controller/nodeipam/ipam/cidrset/cidr_set_test.go

```diff
@@ -198,11 +198,14 @@ func TestIndexToCIDRBlock(t *testing.T) {
 	}
 	for _, tc := range cases {
 		_, clusterCIDR, _ := net.ParseCIDR(tc.clusterCIDRStr)
-		a, err := NewCIDRSet(clusterCIDR, tc.subnetMaskSize)
+		a, err := NewMapCIDRSet(clusterCIDR, tc.subnetMaskSize)
 		if err != nil {
 			t.Fatalf("error for %v ", tc.description)
 		}
-		cidr := a.indexToCIDRBlock(tc.index)
+		ip := a.addIPOffset(tc.index)
```

Comment:
a question here is whether the old method should continue to be unit tested here and new Test... functions should be added for the new code.